src/index.html

```diff
@@ -7,8 +7,84 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a 
+        href="#home" 
+        class="logo"
+      >
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a 
+            href="#apple" 
+            class="nav__link is-active"
+            >
+            Apple
+            </a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a 
              href="#apple" 
              class="nav__link is-active"
            >
              Apple
            </a>
          </li>
```
check the checklist and fix your code style all over the page according to it